packages/swapper/src/swappers/thorchain/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -15,10 +15,11 @@ import { DEFAULT_SLIPPAGE } from '../../utils/constants'
 import { ThorchainSwapperDeps } from '../types'
 import { getThorTxInfo as getBtcThorTxInfo } from '../utils/bitcoin/utils/getThorTxData'
 import { MAX_THORCHAIN_TRADE, THOR_MINIMUM_PADDING } from '../utils/constants'
-import { estimateTradeFee } from '../utils/estimateTradeFee/estimateTradeFee'
+import { estimateBuyAssetTradeFeeCrypto } from '../utils/estimateBuyAssetTradeFeeCrypto/estimateBuyAssetTradeFeeCrypto'
 import { getThorTxInfo as getEthThorTxInfo } from '../utils/ethereum/utils/getThorTxData'
 import { getTradeRate } from '../utils/getTradeRate/getTradeRate'
 import { getUsdRate } from '../utils/getUsdRate/getUsdRate'
+import { isRune } from '../utils/isRune/isRune'
 import { getBtcTxFees } from '../utils/txFeeHelpers/btcTxFees/getBtcTxFees'
 import { getEthTxFees } from '../utils/txFeeHelpers/ethTxFees/getEthTxFees'
 
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"endLine":26,"prLink":"https://app.graphite.dev/github/pr/shapeshift/lib/1043"}} -->
<i class='graphite__hidden'>[Re: line 26]<br/></i>

Note: This is technically correct, but now that the logic actually works as it should for minimums, it is a bit conservative. Given the small discrepancies that can happen with fiat to crypto conversions and back, using a value too close to the minimum as an input will result in reverted swaps.

Probably an edge case as ~1.01$ will fail but ~1.10$ won't, so I will not tackle it in this PR, but I believe we should bump this value a few cents in a follow-up PR @asamere <span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/shapeshift/lib/1043?utm_source=unchanged-line-comment">Graphite</a>.</span>